### DISCUSSION TITLE
Paste multi-paragraph text in a block

### DISCUSSION THREAD

**Discussion by alexlyzhov** [vslinko/obsidian-outliner#264] [Ideas]
Seems like the outliner doesn't affect what's happening when I paste things. As a result, if the pasted text is multi-paragraph, subsequent paragraphs don't have indents and are not included in the block. I paste things to blocks every day and have to add the indentation manually which is suboptimal.

Essentially, the desired behavior for me on Ctrl+V to a block is that all lines of the pasted text starting from the second line are indented so as to ensure they are inside the block.

**Comment by huyz** [comment]
Some related discussion in Obsidian forums: https://forum.obsidian.md/t/better-checkbox-behavior-largely-copy-paste/41296
